operator/pkg/apis/istio/v1alpha2/validation/validation.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validation
```

Comment:
Do we still need to use the old validation for ICP when it is provided as migrate input

Reply:
In principle the input yaml should pass validation, but in practice the translations are very simple so I'm not sure if this check would help users. In any case, this package was not testing much so removing.